hrm-domain/hrm-core/case/sql/case-get-sql.ts

```diff
@@ -25,26 +25,47 @@ export const selectContactsOwnedCount = (ownerVariableName: string) =>
    FROM "Contacts" 
    WHERE "caseId" = cases.id AND "accountSid" = cases."accountSid" AND "twilioWorkerId" = $<${ownerVariableName}>`;
 
-export const selectSingleCaseByIdSql = (tableName: string) => `SELECT
+const leftJoinLateralContacts = (onlyEssentialData?: boolean) => {
+  if (onlyEssentialData) {
+    return `
+      LEFT JOIN LATERAL (
+        SELECT COALESCE(jsonb_agg(to_jsonb(c)), '[]') AS  "connectedContacts"
```

Comment:
We could potentially request less from the contact table here, but this is fine for now, avoiding the extra joins is the real win